libpod/runtime.go

```diff
@@ -1019,7 +1019,7 @@ func (r *Runtime) mergeDBConfig(dbConfig *DBConfig) {
 	if !r.storageSet.GraphDriverNameSet && dbConfig.GraphDriver != "" {
 		if r.storageConfig.GraphDriverName != dbConfig.GraphDriver &&
 			r.storageConfig.GraphDriverName != "" {
-			logrus.Errorf("User-selected graph driver %q overwritten by graph driver %q from database - delete libpod local files to resolve.  May prevent use of images created by other tools",
+			logrus.Errorf("User-selected graph driver %q overwritten by graph driver %q from database - delete libpod local files ($HOME/.local/share/containers) to resolve.  May prevent use of images created by other tools",
```

Comment:
Use `runtime.storageConfig.GraphRoot` instead, it will give the actual storage root in use.